ssg/build_remediations.py

```diff
@@ -146,6 +151,81 @@ def get_populate_replacement(remediation_type, text):
     sys.exit(1)
 
 
+def from_file(file_path, env_yaml):
+    mod_file = []
+    config = defaultdict(lambda: None)
+
+    fix_file_lines = jinja_process_file(file_path, env_yaml).splitlines()
+
+    # Assignment automatically escapes shell characters for XML
+    for line in fix_file_lines:
+        line += "\n"
+        if line.startswith('#'):
+            try:
+                (key, value) = line.strip('#').split('=')
+                if key.strip() in ['complexity', 'disruption',
```

Comment:
The collection of attributes could be defined in a nice constant bearing a descriptive name.